### ISSUE TITLE
vaadinRun task somehow includes slf4j-simple onto the classpath, preventing war-packaged slf4j to log properly.

### ISSUE THREAD

**Issue by mvysny** [johndevs/gradle-vaadin-plugin#371]
Hi, I'm sorry for being annoying :-D Yet I have reproduced the slf4j issue on a really simple project, please find it attached. Just run with `./gradlew clean vaadinRun --info` and observe the following in the console:

```
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/mavi/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-simple/1.7.12/42db62298b899818ff17352cbc00050e940bbfb0/slf4j-simple-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/mavi/.gradle/caches/modules-2/files-2.1/ch.qos.logback/logback-classic/1.1.7/9865cf6994f9ff13fce0bf93f2054ef6c65bb462/logback-classic-1.1.7.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.SimpleLoggerFactory]
```

The slf4j-simple is actually pulled by the vaadin-run-server task: `./gradlew dependencies` shows:
```
vaadin-run-server - Libraries for running the embedded server
...
+--- org.slf4j:slf4j-simple:1.7.12
|    \--- org.slf4j:slf4j-api:1.7.12
```

Is it possible to remove this dependency please?
[vaadingradletest.zip](https://github.com/johndevs/gradle-vaadin-plugin/files/915564/vaadingradletest.zip)

**Comment by johndevs** [comment]
> Hi, I'm sorry for being annoying :-D
No worries ;D

> SLF4J: Found binding in [jar:file:/home/mavi/.gradle/caches/modules-2/files-2.1/ch.qos.logback/logback-classic/1.1.7/9865cf6994f9ff13fce0bf93f2054ef6c65bb462/logback-classic-1.1.7.jar!/org/slf4j/impl/StaticLoggerBinder.class]

I think this is coming from Gretty. Maybe if you remove that then vaadinRun should work?

But I'll look into why Jetty is pulling in slf4J, I don't see any immediate reason why it would be needed.

**Comment by mvysny** [comment]
Thanks! Actually, in this project the "gretty" plugin is commented out so it shouldn't be used at all. Omitting the comments, this is how the `build.gradle` file looks like:

```
plugins {
  id 'fi.jasoft.plugin.vaadin' version '1.1.8'
}

vaadin {
    version '8.0.5'
    logToConsole = true
    vaadinRun {
        server = 'jetty'
    }
}
dependencies {
    compile 'ch.qos.logback:logback-classic:1.1.7'
    compile 'org.slf4j:slf4j-api:1.7.21'
}
```

**Comment by mvysny** [comment]
Using 1.1.9 will now print the following to the console:
```
SLF4J: The requested version 1.7.16 by your slf4j binding is not compatible with [1.6]
SLF4J: See http://www.slf4j.org/codes.html#version_mismatch for further details.
```
Now the real dependency is unmasked - when you upgrade to 1.1.9 and run `./gradlew dependencies`, it is clear that jetty now depends on slf4j:
```
vaadin-run-server - Libraries for running the embedded server
...
+--- org.eclipse.jetty.aggregate:jetty-all:9.3.6.v20151106
...
|    \--- org.slf4j:slf4j-api:1.6.6
...
```
I'm not sure why Jetty-all depends on slf4j, but maybe the dependency can be excluded?

**Comment by johndevs** [comment]
@mvysny This was classpath priorizitation issue. I've fixed that in #376 and should work in ``1.1.10`` once it is released (at least your test project works with it).
